src/components/header/header.e2e.ts

```diff
@@ -0,0 +1,56 @@
+import { newE2EPage } from '@stencil/core/testing';
+
+describe('c-header', () => {
+  let page;
+  const root = 'c-header >>>';
+  const siteName = 'Application';
+
+  beforeEach(async () => {
+    page = await newE2EPage();
+  });
+
+  it('is rendered', async () => {
+    await page.setContent(`
+      <c-header site-name="${siteName}"></c-header>`);
+
+    const symbol = await page.find(`${root} .navbar-symbol`);
+    expect(symbol).not.toBeNull();
+
+    const navbar = await page.find(`${root} .navbar-default`);

```

Comment:
If we check that the ".navbar-default" class is existing, then we should add a check for ".navbar".

This is something for later in the testing development